coloringbook/admin/views.py

```diff
@@ -326,6 +326,60 @@ class SurveyView(ModelView):
         'email_address': 'Used to send a notification when a survey is completed and uploaded.',
     }
 
+    @action('duplicate', 'Duplicate', 'Are you sure you want to duplicate the selected surveys?')
```

Comment:
I suggest not asking for confirmation. It does not seem very likely that someone will do this by accident, but if it happens, it is still non-destructive. Users are generally annoyed by having to click through confirmations.